src/MySql/custom/New-AzMySqlFirewallRule.ps1

```diff
@@ -0,0 +1,142 @@
+
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.Synopsis
+Creates a new firewall rule or updates an existing firewall rule.
+.Description
+Creates a new firewall rule or updates an existing firewall rule.
+
+.Link
+https://docs.microsoft.com/en-us/powershell/module/az.mysql/new-azmysqlfirewallrule
```

Comment:
URL's should not include culture.
```suggestion
https://docs.microsoft.com/powershell/module/az.mysql/new-azmysqlfirewallrule
```

Reply:
Link should be auto generated. It is added by mistake.  We will fix it in powershell generator.